src/sentry/static/sentry/app/index.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta http-equiv="content-type" content="text/html; charset=utf-8" />
+    <meta name="robots" content="NONE,NOARCHIVE" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+
+    <link href="/_assets/sentry.css" rel="stylesheet" />
```

Comment:
Can we use the html plugin?

Reply:
Yeah we can, but we don't really get too much out of it right now. It also can't pull out assets from the legacy sass stuff, since we have a multi-target build, so that would end up needing to be hard coded for now (or I could look closer at how to get it to include multi target outputs).

I'd be okay investigating later though.